src/app/frontend/index_module.js

```diff
@@ -35,6 +36,7 @@ export default angular.module(
                             'ngSanitize',
                             'ui.router',
                             chromeModule.name,
+                            commonsModule.name,
```

Comment:
Index does not depend on commons module, deploy module does. Dependencies should be declared in the places where they are actually used. Otherwise we would end up with one catch-all module listing all dependencies, which voids the idea of reusability.